usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -78,5 +78,7 @@ def return_query_strings(use_matview):
     if use_matview:
         q_str = '{0}_{1}__in'
         loc_dict['country'] = 'country_code'
+    else:
+        loc_dict['country'] = 'location_country_code'
```

Comment:
what does `location_country_code` refer to here?

Reply:
It refers to what the country code column name in the table the function is working from. Matviews use `country_code` while baby download uses the transaction filter, not using the matview, so it goes through the location table which uses `location_country_code`. @shinson  was the one who discovered the fix and can explain it better. Without said fix, we found some of the baby downloads were returning 500s.

Reply:
Look at lines 5-6 on that file. `location_country_code` is the default value for loc_dict['country'] and doesn't need to be set.